language/extensions/move-table-extension/Move.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "MoveTableExtension"
+version = "1.0.0"
+
+[addresses]
+Std = "0x1"
+Extensions = "0x2"
```

Comment:
Should these be in `[dev-addresses]`? Or is it required for now that it both are at these specific addresses?

Reply:
Done.